src/Components/Header/Header.js

```diff
@@ -19,6 +24,15 @@ const Header = () => {
           contact
         </NavLink>
       </section>
+      <section className="header-hamburger">
+        <Hamburger
+          onClick={openModal}
+          onKeyDown={openModal}
```

Comment:
The onKeyDown attribute is needed to make this accessible - if someone uses the keyboard to navigate to the hamburger icon, it should still open the menu.